ormin/ormin_postgre.nim

```diff
@@ -25,13 +25,15 @@ proc c_strtod(buf: cstring, endptr: ptr cstring = nil): float64 {.
 proc c_strtol(buf: cstring, endptr: ptr cstring = nil, base: cint = 10): int {.
   importc: "strtol", header: "<stdlib.h>", noSideEffect.}
 
-var sid {.compileTime.}: int
+var sid = 0
+
+template genSid*: untyped =
+  inc(sid)
+  sid
 
 proc prepareStmt*(db: DbConn; q: string): PStmt =
-  static:
```

Comment:
Seems worse than my solution. Why is it required?

Reply:
actually this does not work, the constant name does not exists when the rest of the proc is executed giving this error message: 

ormin/ormin/ormin_postgre.nim(35, 20) Error: undeclared identifier: 'name'